core/jobs.py

```diff
@@ -720,6 +958,7 @@ def get(cls, entity_id, strict=True):
         ).get(entity_id, strict=strict)
 
     def put(self):
+        """TODO(brianrodri): This method seems unneccesary."""
```

Comment:
Line 405 is for VersionedModel, not BaseModel. So I think the code is correct as-is...